kolibri/content/api.py

```diff
@@ -255,10 +257,11 @@ def prefetch_related(self, queryset):
         return queryset.prefetch_related(
             'assessmentmetadata',
             'files',
+            'files__local_file'
         ).select_related('license')
 
     def get_queryset(self, prefetch=True):
-        queryset = models.ContentNode.objects.all()
+        queryset = models.ContentNode.objects.filter(available=True)
```

Comment:
What happened that we now have to filter by `available`?

Reply:
We actually mark whether content is available or not, rather than just assuming it is all there. This will handle incomplete imports more gracefully.